src/gui_common/QuickLoadHandler.tscn

```diff
@@ -10,10 +11,12 @@ script = ExtResource( 1 )
 __meta__ = {
 "_edit_use_anchors_": false
 }
-DifferentVersionDialogPath = NodePath("DifferentVersionDialog")
+DifferentVersionDialogPath = NodePath("CustomConfirmationDialog")
```

Comment:
Why was this node renamed? It used to be specific to what it actually did, instead of just being the class name.

Reply:
Must have forgot to rename this...